src/goto-symex/rewrite_union.cpp

```diff
@@ -20,76 +20,68 @@ Author: Daniel Kroening, kroening@kroening.com
 
 #include <util/c_types.h>
 
-static bool have_to_rewrite_union(
-  const exprt &expr,
-  const namespacet &ns)
+static bool have_to_rewrite_union(const exprt &expr)
```

Comment:
This commit should really be in a separate pull request. It's likely the second commit that breaks things, while this one ought to have no impact at all. Also: the commit message could be much clearer because all that happens is removing the unused namespace parameter.